main.go

```diff
@@ -455,6 +458,38 @@ func mainErr(args []string) error {
 	transformed := args[1:]
 	// log.Println(tool, transformed)
 	if transform != nil {
+		if len(args) == 2 && args[1] == "-V=full" {
+			cmd := exec.Command(args[0], args[1:]...)
+			out, err := cmd.Output()
+			if err != nil {
+				if err, _ := err.(*exec.ExitError); err != nil {
```

Comment:
Why not `if err, ok := err.(*exec.ExitError); ok {`

Reply:
It's pretty equivalent, really, unless it's possible for a function to return a nil ExitError. The advantage of doing it this way is that you elide nil checks, in the if body, because that's your if condition. For the same reason, it's impossible for the if body to panic with a nil pointer dereference.

It doesn't really matter here, it's just something I've gotten used to. Either way would be fine I think.